.github/workflows/publish-halmos-builder-package.yml

```diff
@@ -4,6 +4,10 @@ name: Push halmos-builder package
 on:
   # only trigger manually, this is independent from halmos releases
   workflow_dispatch:
+  workflow_run:
+    workflows: ["Push solvers package"]
+    types:
+      - completed
```

Comment:
works, this was trigger by a successful run of publish-solvers-package.yml: https://github.com/a16z/halmos/actions/runs/11751217641